### PR TITLE
UI Fit & Finish Pre-check Changes

### PR DESCRIPTION
**Pull request by davidcui-amzn**
*Issue #, if available:*
N/A
*Description of changes:*
UI Changes added from the fit and finish pre-check 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by anirudha — APPROVED** [review]

File: kibana-reports/public/components/report_definitions/report_settings/report_settings.tsx [hunk 1]
```diff
@@ -118,9 +121,10 @@ function TimeRangeSelect() {
     <div>
       <EuiFormRow
         label="Time range"
-        helpText="Time range is relative to the report creation date on the report trigger"
+        helpText="Time range is relative to the report creation date on the report trigger."
       >
         <EuiSuperDatePicker
+          isDisabled={false}
```
Comment (joshuali925) [inline kibana-reports/public/components/report_definitions/report_settings/report_settings.tsx:127]: Why is this always disabled?

File: kibana-reports/public/components/main/report_definitions_table.tsx [hunk 2]
```diff
@@ -78,7 +33,9 @@ const emptyMessageReportDefinitions = (
         <EuiText>Create a new report definition to get started</EuiText>
         <EuiText>
           To learn more, see{' '}
-          <EuiLink>Get started with Kibana reporting</EuiLink>
+          <EuiLink>
```
Comment (joshuali925) [inline kibana-reports/public/components/main/report_definitions_table.tsx:36]: Some of those EuiLinks don't seem to do anything, is this on purpose?

**Review by joshuali925 — APPROVED** [review]
LGTM, just some questions

**Review by akbhatta — APPROVED** [review]

File: kibana-reports/public/components/main/report_definitions_table.tsx (hunk 2 above)
Comment (davidcui-amzn) [reply kibana-reports/public/components/main/report_definitions_table.tsx:36]: They are placeholder links for now to render the UI- they will be populated once the pages are running (e.g the Reporting documentation page)

**Pull request merged**